kernel/src/arch/x86_64/mod.rs

```diff
@@ -512,15 +531,12 @@ fn _start(argc: isize, _argv: *const *const u8) -> isize {
             crate::nrproc::register_thread_with_process_replicas();
```

Comment:
Previously, the controller initialized the log but didn't have a replica of it's own, so the total number of process replicas was equal to the number of clients. Reading the code, I believe this is still true, but I wanted to confirm.